test/squish/suite_VerifySettingsDialog/shared/scripts/gui_helpers.py

```diff
@@ -0,0 +1,181 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*
+# mypy: disable-error-code="attr-defined"
+
+import gui_locators
+import squish
+import test
+
+snoozeTimeout = 3  # seconds
+
+
+def launchDeadlineConfigGUI():
+    squish.startApplication("deadline config gui")
+    test.log("Launching Deadline Config GUI")
+    test.log(
+        "Sleep for " + str(snoozeTimeout) + " seconds to allow GUI authentication to fully load."
+    )
+    squish.snooze(snoozeTimeout)
+    test.compare(
```

Comment:
By 'state of the application', do you mean to whether the application is opened, or not?

Reply:
Ok - I think I misunderstood the comment. I guess when I worked in Selenium, these functions were called Page Objects. I don't know what's the equivalent name for GUI QT testing.....

Reply:
Since we are running tests in these functions, I wouldn't consider them helpers but the tests themselves. I expect utility functions to be in a file called `gui_helpers.py`